docs/sidebars.js

```diff
@@ -40,6 +40,11 @@ const sidebars = {
       label: "Image",
       id: "image",
     },
+    {
```

Comment:
This doesn't seem to fit here. Should just document how to do it via the reconciler for now? (theoretically the imperative documentation can already be found on Flutter or CanvasKit (more or less)).